core/src/server/DBWrapper.cpp

```diff
@@ -54,21 +54,24 @@ DBWrapper::StartService() {
     std::string db_slave_path;
     s = config.GetDBConfigSecondaryPath(db_slave_path);
     if (!s.ok()) {
-        return s;
+        std::cerr << s.ToString() << std::endl;
+        kill(0, SIGUSR1);
```

Comment:
Here "return s" is for specific reason, only do so can let system print out "Milvus server exit..." and tell user Milvus server does not boot up successfully.
If use "kill()", system will exit quitely without any message.